client/src/App.vue

```diff
@@ -31,7 +30,10 @@ export default {
   },
 
   created () {
-    if (Authorized.isAuthorized()) {
+    if (
+      window.localStorage.getItem('token') &&
+      window.localStorage.getItem('token') !== ''
```

Comment:
Why you remove isAuthorized? And why you need extra check for window.localStorage.getItem('token') !== ''?